modules/core/arrow-test/src/main/kotlin/arrow/test/laws/BimonadLaws.kt

```diff
@@ -8,27 +8,35 @@ import io.kotlintest.properties.forAll
 
 object BimonadLaws {
   fun <F> laws(BF: Bimonad<F>,
-               f1: (Int) -> Kind<F, Int>,
+               f: (Int) -> Kind<F, Kind<F, Int>>,
                EQ1: Eq<Int>,
-               f2: (Int) -> Kind<F, Kind<F, Int>>,
                EQ2: Eq<Kind<F, Kind<F, Int>>>): List<Law> =
     listOf(
```

Comment:
This should be MonadLaws + ComonadLaws + listOf

Reply:
Currently, with your suggestion, BimonadLaws would be unnecessary because in every datatype we already have that exact setting. What I wanted to introduce here is that the laws work in conjunction, and not solely in parallel, as before. 

That's why the `coflatMap` composition with `just`.

Reply:
>BimonadLaws would be unnecessary because in every datatype we already have that exact setting

It should replace it, and extend it with the additional laws below!

Reply:
For example, something that's Async and Comonad may not need the AsyncLaws check removed, but the Comonad one can be replaced with Bimonad.

Reply:
agree